guides/source/getting_started.md

```diff
@@ -433,7 +433,7 @@ If you refresh <http://localhost:3000/articles/new> now, you'll get a new error:
 
 This error indicates that Rails cannot find the `new` action inside the
 `ArticlesController` that you just generated. This is because when controllers
-are generated in Rails they are empty by default, unless you tell it
+are generated in Rails they are empty by default unless you tell it
```

Comment:
I double checked. A comma is not needed before `unless` unless the sentence is very long/complex.